packages/contracts-bedrock/scripts/deploy/DeployAltDA2.s.sol

```diff
@@ -92,7 +92,6 @@ contract DeployAltDA2 is Script {
         require(_input.challengeWindow != 0, "DeployAltDA: challengeWindow not set");
         require(_input.resolveWindow != 0, "DeployAltDA: resolveWindow not set");
         require(_input.bondSize != 0, "DeployAltDA: bondSize not set");
-        require(_input.resolverRefundPercentage != 0, "DeployAltDA: resolverRefundPercentage not set");
```

Comment:
Assuming that `_input.resolverRefundPercentage=0` is actually a valid input since some op-e2e tests use that input with the legacy `DeployAltDA` script